pkg/volume/vspherevolume/attacher_test.go

```diff
@@ -266,8 +266,8 @@ func (testcase *testcase) DetachDisk(diskName string, nodeName types.NodeName) e
 	if expected.diskName == "" && expected.nodeName == "" {
 		// testcase.detach looks uninitialized, test did not expect to call
 		// DetachDisk
-		testcase.t.Errorf("Unexpected DetachDisk call!")
-		return errors.New("Unexpected DetachDisk call!")
+		testcase.t.Errorf("Unexpected DetachDisk call")
+		return errors.New("Unexpected DetachDisk call")
```

Comment:
Are these `testcase.t.Errorf` calls intended? To me they seem like some boilerplate that is not required since the test function will check for errors again: https://github.com/kubernetes/kubernetes/pull/91687/files#diff-fe4bc2e0131f8921ad973dd8f259a136L157?

Reply:
I'm not sure about it. Which test function do you mean?

Reply:
Line 157 of this file (somehow the link doesn't work)